src/app.js

```diff
@@ -694,7 +707,9 @@ function run () {
     app: self.event,
     udapp: udapp.event
   }
-  var righthandPanel = new RighthandPanel(document.body, rhpAPI, rhpEvents, {}) // eslint-disable-line
+  var righthandPanel = new RighthandPanel(rhpAPI, rhpEvents, {}) // eslint-disable-line
+  self._view.rightpanel.replaceWith(righthandPanel.render())
```

Comment:
Please just make `self._view.rightpanel` a container that supports resizing (or will support in the next PR) and `RightHandpanel` a module that just fit it in his container (and thus follow the size of the container).